### PR TITLE
feat: convert relative urls to absolute

### PR DESCRIPTION
**Pull request by mrtwnklr**
Origin: see #41 

Completion of relative urls is configurable with `enable-url-completion`. Default is `false`.
For known issues see changes in [README.md](https://github.com/peter-evans/dockerhub-description/pull/125/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

For test cases see:
- this markdown file: https://github.com/mrtwnklr/workflow-tests/blob/main/DOCKERHUB_DESCRIPTION.md
- this generated description: https://hub.docker.com/r/mrtwnklr/workflow-tests

### CODE REVIEW

File: README.md
Comment (peter-evans) [inline README.md:28]:
```suggestion
| `enable-url-completion` | Enables completion of relative URLs to absolute ones. See also [Known Issues](#known-issues). | `false` |
```

File: README.md
Comment (peter-evans) [inline README.md:29]:
```suggestion
| `image-extensions` | File extensions that will be treated as images. | `bmp,gif,jpg,jpeg,png,svg,webp` |
```

**Review by peter-evans — COMMENTED** [review]
This is great. Thank you!

Would you be willing to codify the test cases you have here?
https://github.com/mrtwnklr/workflow-tests/blob/main/DOCKERHUB_DESCRIPTION.md#test-cases

This project doesn't have unit tests yet because it wasn't really that necessary until now. I think `readme-helper` should really have unit tests, though.

Please put them under `__test__`. e.g.,
https://github.com/peter-evans/find-comment/tree/main/__test__

And just remove `--passWithNoTests` here:
https://github.com/peter-evans/dockerhub-description/blob/main/package.json#L12

**Comment by mrtwnklr** [comment]
> Would you be willing to codify the test cases you have here? https://github.com/mrtwnklr/workflow-tests/blob/main/DOCKERHUB_DESCRIPTION.md#test-cases

Now that you're asking, of course.

**Comment by mrtwnklr** [comment]
:ballot_box_with_check: tests

**Comment by peter-evans** [comment]
Please run `npm run format` to fix the format. Or you can "allow edits from the maintainer" on the PR.

**Comment by mrtwnklr** [comment]
sry, done

**Review by peter-evans — APPROVED** [review]
Thank you for contributing!

I'm going to merge to a `dev` branch before merging to main because some of my CI tests cannot run on pull requests from forks.

**Comment by peter-evans** [comment]
Released as `v3.2.0` / `v3`.

**Comment by mrtwnklr** [comment]
Great! Maybe worth closing #41 ?

**Comment by peter-evans** [comment]
I left a comment here: https://github.com/peter-evans/dockerhub-description/issues/41#issuecomment-1455352009

I just want to check if the emoji feature is feasible or not before deciding whether to close.

**Pull request merged**
